packages/manager/modules/hub/src/components/support/template.html

```diff
@@ -0,0 +1,66 @@
+<div class="manager-hub-support" data-ng-if="!$ctrl.isLoading">
+    <div data-ng-if="$ctrl.tickets.length > 0">
+        <h3 data-translate="hub_support_title"></h3>
```

Comment:
No `h2` above?

Reply:
@FredericEspiau the `h2` will be brought by the component's parent when integrated I guess. 
Imo it's quite complicated because components shouldn't always have a `h1` but it's hard to know if the headings hierarchy will be valid

Reply:
Yes this is why I ask if there is a `h2` above because you neven know with `html`

At some point, to fix this, `html` implemented an outline algorithm but it didn't work properly so nobody actually implemented it: https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Using_HTML_sections_and_outlines